### PR TITLE
Implement createDigestString() method.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Extracted from https://github.com/digitalbazaar/http-signature-zcap-invoke/blob/master/main.js#L35

WIP, todo:
- [ ] Find an isomorphic way to base64-encode strings (our `base64url-universal` does not expose non-url encoding)
- [ ] Find some test strings for multihash digest?

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
Hmmm except `isomorphic-webcrypto` requires Node 12. Maybe I need to put this into a separate npm package?

File: bin/util.js [hunk 1]
```diff
@@ -95,15 +95,15 @@ function validatePrivateKey(keyObj) {
  * @param {Array<string>} options.includeHeaders - Which headers
  * to use in the Signing String.
  *
- * @returns {Object} The response headers.
+ * @returns {object} The response headers.
```
Comment (aljones15) [inline bin/util.js]: thanks for linting this.

File: test/test.js [hunk 2]
```diff
@@ -12,6 +12,16 @@ const HttpSignatureError = require('../lib/HttpSignatureError');
 
 describe('http-signature', () => {
 
+  describe('createDigestString', () => {
```
Comment (aljones15) [inline test/test.js]: thanks for the test.

**Comment by aljones15** [comment]
@dlongley  this does not break any existing http-signatures-test-suite tests (except for the already failing ones for created and expires). just not sure on if we want to use `isomorphic-webcrypto`

**Comment by dlongley** [comment]
-1 to this PR without some design changes. We've avoided putting any crypto stuff into this library thus far and I'd like us to explore continuing that. The simplest approach would be to just put this new stuff into its own module, like http-signature-digest-header or similarly named.

**Comment by dmitrizagidulin** [comment]
@dlongley ok, copy that, I wasn't sure about that part.
I'll move it to a different repo.

**Comment by dlongley** [comment]
@dmitrizagidulin -- Thanks! Sorry about that.

**Comment by dmitrizagidulin** [comment]
Extracted to https://github.com/digitalbazaar/http-signature-digest/pull/1, closing.

**Pull request closed without merging**